xcodeproj/internal/output_group_map.bzl

```diff
@@ -0,0 +1,161 @@
+"""Module containing functions dealing with output group map files."""
+
+# Each "file" might actually represent many files (if the "file" is an output
+# directory). This value was optimized around indexstores. The larger this value
```

Comment:
What exactly is the connection with indexstores here (or is there none and it’s just from your past experience)?

Reply:
Past experience. This code can be used for fetching indexstores in the future though, in Build with Bazel mode, though it isn't right now.